### PR TITLE
Made renaming refactoring to make it more readable.

### PR DESCRIPTION
**Pull request by nurolopher**

### CODE REVIEW

File: lib/PhpParser/Builder/Class_.php [hunk 1]
```diff
@@ -43,10 +43,8 @@ public function extend($class) {
 
     /**
      * Implements one or more interfaces.
-     *
-     * @param Name|string ...$interfaces Names of interfaces to implement
```
Comment (GrahamCampbell) [inline lib/PhpParser/Builder/Class_.php]: ?

File: lib/PhpParser/Builder/Declaration.php [hunk 2]
```diff
@@ -10,7 +10,7 @@
 {
     protected $attributes = array();
 
-    abstract public function addStmt($stmt);
+    abstract public function addStatement($statement);
```
Comment (GrahamCampbell) [inline lib/PhpParser/Builder/Declaration.php]: this is a major breaking change

File: lib/PhpParser/Builder/Interface_.php [hunk 3]
```diff
@@ -40,23 +40,23 @@ public function extend() {
...
-                $this->constants[] = $stmt;
+                $this->constants[] = $statement;
                 break;
 
             case 'Stmt_ClassMethod':
                 // we erase all statements in the body of an interface method
-                $stmt->stmts = null;
-                $this->methods[] = $stmt;
+                $statement->stmts = null;
+                $this->methods[]  = $statement;
```
Comment (GrahamCampbell) [inline lib/PhpParser/Builder/Interface_.php]: this change doesn't make sense

File: test/PhpParser/BuilderFactoryTest.php [hunk 4]
```diff
@@ -37,35 +37,35 @@ public function testNonExistingMethod() {
...
                                       *
                                       * @param SomeClass And takes a parameter
                                       */'))
 
-                ->addStmt($factory->method('anotherMethod')
-                    ->makeProtected()
-                    ->addParam($factory->param('someParam')->setDefault('test'))
-                    ->addStmt(new Expr\Print_(new Expr\Variable('someParam'))))
+                ->addStatement($factory->method('anotherMethod')
+                                       ->makeProtected()
```
Comment (GrahamCampbell) [inline test/PhpParser/BuilderFactoryTest.php]: these alignment changes don't make sense?

Comment (GrahamCampbell) [inline test/PhpParser/BuilderFactoryTest.php]: the code style of indenting by 4 spaces for each level is enforced here, but your changes break that

**Comment by GrahamCampbell** [comment]
Made some comments. I'm :-1: on the entire PR actually.

**Comment by nurolopher** [comment]
@GrahamCampbell after reading your comments I also don't think it's a good pull request! Thank you for the feedback.

**Pull request closed without merging**